cli/worker.rs

```diff
@@ -67,7 +67,6 @@ impl CliMainWorker {
     log::debug!("main_module {}", self.main_module);
 
     if self.is_main_cjs {
```

Comment:
It might be an independent issue, but if the script only contains `node:` specifier, this flag seems to become `false` and `initialize_main_module_for_node()` isn't called.

The below script works fine:
```ts
import express from "npm:express";
import process from "node:process";
console.log(process.argv);
```

but the below doesn't have correct `process.argv`
```ts
import process from "node:process";
console.log(process.argv);
```

Reply:
https://github.com/denoland/deno/blob/4104a674c7955eb7811b241b2e18453ac96faaf5/cli/worker.rs#L467-L468

I think this needs to also match for `node::NodeResolution::Builtin(..)`

Reply:
Good catch Yoshiya. I can see what's going wrong here.

@littledivy not really, `is_main_cjs` describes if the `deno run <script_name>` is a CJS file or not. The problem is that I called `internal.__bootstrapNodeProcess()` only when initializing CJS runtime. I will improve tests and fix the problem before landing this PR.

Reply:
Fixed in https://github.com/denoland/deno/pull/17785/commits/4f8cf07386624a882ce0e56af321b0c51f8c7383